app/controllers/admin/claim_rejections_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Admin::ClaimRejectionsController < Admin::BaseAdminController
+  before_action :ensure_service_operator
+
+  def create
```

Comment:
Thinking about this, I'm wondering whether the first step in the rejection process should be a `new` action, which renders a form where a user can add their reasoning, which then submits to the `create` action

Reply:
Yeah, it's okay to take this is steps though right?

Reply:
Oh yeah, totally. Was just idle musing at this stage 😄

Reply:
nice! :D

Reply:
Is there a point where the `reject` and `approve` controllers become a single `actions` controller? Are the advantages/disadvantages to the approaches?

Reply:
Yeah, I think that's the point where we have everything wrapped into a single validation model, but I haven't quite got it straight in my head as to how that'll work